src/main/java/edu/cornell/kfs/module/cam/rest/resource/CuCapAssetInventoryApiResource.java

```diff
@@ -184,18 +189,45 @@ public Response updateAsset(@PathParam(CuCamsConstants.CapAssetApi.ASSET_TAG_PAR
             return ex instanceof BadRequestException ? respondBadRequest() : respondInternalServerError(ex);
         }
     }
+    
+    @GET
+    @Path("/campusCodes")
+    public Response getCampusCodes(@Context HttpHeaders headers) {
+        try {
+            List<Map<String, String>> campusCodes = new ArrayList<>();
+            
+            Map<String, Object> fieldValues = new HashMap<>();
+            fieldValues.put(CuCamsConstants.CapAssetApi.ACTIVE, true);
+            Collection<Campus> campuses = getBusinessObjectService().findMatching(Campus.class, fieldValues);
+            
+            for (Campus campus : campuses) {
+                Map<String, String> campusMap = new HashMap<>();
+                campusMap.put("value", campus.getCode());
+
+                String campusLabel = String.format("%s - %s", campus.getCode(), campus.getName());
+                campusMap.put("label", campusLabel);
```

Comment:
You may want to use constants for both the map keys and the format string.

Reply:
Please see my latest changes